snooty/postprocess.py

```diff
@@ -677,6 +681,16 @@ def exit_page(self, fileid_stack: FileIdStack, page: Page) -> None:
                     for tabid, title in tabsets[0].items()
                 }
 
+                # If default_tabs are present, append to page options
+                if tabset_name in self.default_tabs:
+                    if not page.ast.options.get("default_tabs"):
+                        page.ast.options["default_tabs"] = {}
+
+                    assert isinstance(page.ast.options["default_tabs"], Dict)
+                    page.ast.options["default_tabs"][tabset_name] = self.default_tabs[
```

Comment:
curious why we are making this a page option than an option on the resulting directive

Reply:
If I were to guess, it might be for convenience? If a page has multiple instances of `tabs-drivers` found on different parts of the page, then a writer would have to make sure every instance has the same default.

Reply:
Wait, I misread the original question. Thought this was about `tabs`. Ignore me 😅

Reply:
I was working a bit backwards too, because the tab-context in Snooty uses the page's ast options to decide the default tab and the options available. 

Both tab-selectors and tab-pillstrip are parent directives. Therefore, it's a page-wide directive? Tell me if I'm completely wrong.

Reply:
ah i see, i guess this would be a page option since each (driver) tab on this page should be the same value anyway. i was thinking in the general realm of tabs if we can have a default value option, but for tabs selectors it should be consistent across the page 👍